OracleGoldenGate/Dockerfile

```diff
@@ -0,0 +1,79 @@
+# LICENSE CDDL 1.0 + GPL 2.0
+#
+# Copyright (c) 1982-2017 Oracle and/or its affiliates. All rights reserved.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is a Dockerfile for Oracle GoldenGate 12c Release 1.   It depends upon the 
+# Docker image for Oracle Database at: 
+# https://github.com/oracle/docker-images/tree/master/OracleDatabase.
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# (1) fbo_ggs_Linux_x64_shiphome.zip
+#     
+#     Download Oracle GoldenGate 12c Release 1 Enterprise Edition for Linux x64
+#     from http://www.oracle.com/technetwork/middleware/goldengate/downloads/index.html
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Put all downloaded files in the same directory as this Dockerfile
+# Run:
+#      $ docker build -t 
+#
+# Pull base image
+# ---------------
+
+FROM oracle/database:12.1.0.2-ee
```

Comment:
Current GoldenGate 12.3 is not GA which supports DB 12.2EE yet so I'm just waiting for that to be available before I provide support for it for capture and apply.